SU-SWS/saa_victoria_subtheme
# Adapt 349  fonts

# READY FOR REVIEW

# Summary
- Add basic fonts mixins and apply to headings

# Review By (Date)
- 6/10 if possible, 

# Urgency
- helpful to have before the next style tasks

# Steps to Test

1. pull down theme branch
1. clear cache on your demo page and see headings in cards and banners have calluna font and font-weigh 900 etc.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADAPT-349
- Helpful for all the next style tasks going forward

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
